lib/torrent.js

```diff
@@ -951,6 +951,7 @@ class Torrent extends EventEmitter {
   }
 
   removePeer (peer) {
+    if (!this._peers) return
```

Comment:
```suggestion
    if (!this._peers) throw new Error('torrent is destroyed')
```

Reply:
this will error when PeX is closing/cleaning up, or when PeX send a drop peer request as the torrent is being destroyed
https://github.com/webtorrent/webtorrent/blob/master/lib/torrent.js#L1098

Reply:
also this is just relocating the error, as webtorrent iself would throw the error instead of erroring 2 lines later on `peer = this._peers[id]` aka what this is trying to fix